validator/client/validation_client.go

```diff
@@ -145,13 +146,37 @@ func (c *ValidationClient) Room() int {
 
 type ExecutionClient struct {
 	ValidationClient
+	boldValClient *redis.BoldValidationClient
 }
 
-func NewExecutionClient(config rpcclient.ClientConfigFetcher, stack *node.Node) *ExecutionClient {
+func NewExecutionClient(config rpcclient.ClientConfigFetcher, redisBoldValidationClientConfig *redis.ValidationClientConfig, stack *node.Node) *ExecutionClient {
```

Comment:
boldValclient does not need to be a field of executionclient, they are separate beings.